lambda-notes/src/Components/rightContent/MyNotes.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MyNotes = (props) => {
+	return (
+		<div>
+			{props.notes.map((note) => {
+				return (
+					<NoteDetails key={note._id} title={note.title} textBody={note.textBody} _id={note._id} {...props} />
```

Comment:
Since you're not using {note._id} in the NoteDetails component, you don't necessarily need to send it down as a prop here.